build-plugins/copy-types.ts

```diff
@@ -16,3 +16,24 @@ export default function copyTypes(fileName: string): Plugin {
 		name: 'copy-types'
 	};
 }
+
```

Comment:
As `copyTypes` is now only used for the browser types, maybe rename it to `copyBrowserTypes` and remove/hard code the `fileName` parameter to `rollup.browser.d.ts`.

Then `copyAllTypes` could be renamed `copyNodeTypes`

Reply:
Ok, will do.

Reply:
I changed the name of `copyTypes` to `copyRollupType` and added a new `copyBrowserTypes ` function.